Userland/Libraries/LibWeb/Layout/TableFormattingContext.cpp

```diff
@@ -184,6 +184,8 @@ void TableFormattingContext::compute_outer_content_sizes()
             auto min_width = computed_values.min_width().to_px(column_box, containing_block.content_width());
             auto max_width = computed_values.max_width().is_length() ? computed_values.max_width().to_px(column_box, containing_block.content_width()) : CSSPixels::max();
             auto width = computed_values.width().to_px(column_box, containing_block.content_width());
+            if (column_index >= m_columns.size())
+                return;
```

Comment:
it seems that this check hides a bug in the columns iteration process: column_index should never be greater than columns vector size.